src/browser/builds/algoliasearchLite.js

```diff
@@ -3,4 +3,4 @@
 var AlgoliaSearchCore = require('../../AlgoliaSearchCore.js');
 var createAlgoliasearch = require('../createAlgoliasearch.js');
 
-module.exports = createAlgoliasearch(AlgoliaSearchCore, '(lite) ');
+module.exports = createAlgoliasearch(AlgoliaSearchCore, 'lite ');
```

Comment:
The trailing space was already present, but is it necessary? The jQuery one e.g. doesn't seem to have it

Reply:
Ok I understand, it's to avoid the double space